docs/source/tracking.rst

```diff
@@ -71,7 +71,9 @@ call :py:func:`mlflow.set_tracking_uri`.
 There are different kinds of remote tracking URIs:
 
 - Local file path (specified as ``file:/my/local/dir``), where data is just directly stored locally.
-- Database encoded as a connection string (specified as ``db_type://<user_name>:<password>@<host>:<port>/<database_name>``)
+- Database encoded as ``<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>``.
+Mlflow supports the dialects ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``. For more
+details refer `SQLAlchemy database uri <https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls>`_
 - HTTP server (specified as ``https://my-server:5000``), which is a server hosting an :ref:`MLFlow tracking server <tracking_server>`.
```

Comment:
For more details refer SQLAlchemy database uri -> For more details, see SQLAlchemy database URI